script.js

```diff
@@ -1,76 +1,221 @@
-    "<br><br>" +
-    "Or add 'reverse' before your input to try something different."
-  );
+    //Returns result to the player
+    if (gameType == "normal") {
+      return (
+        "You chose " +
+        userChoice +
+        "." +
+        "<br><br>" +
+        "The bot chose " +
+        botChoice +
+        "." +
+        "<br><br>" +
+        result +
+        "<br><br>" +
+        "Now you can type 'scissors', 'paper', or 'stone' to play another round!" +
+        "<br><br>" +
+        "Or add 'reverse' before your input to try something different." +
+        "<br><br>" +
+        "Hey, " +
+        userName +
+        " you have played " +
+        gameRecord +
+        " games so far. Your win:loss:draw figures is " +
+        winRecord +
+        ":" +
+        lossRecord +
+        ":" +
+        drawRecord +
+        "." +
+        performanceMesssage +
+        "."
+      );
+    } // Computer mode return
+    else if (gameType == "computer") {
+      return (
+        "Your bot chose " +
+        userChoice +
+        "." +
+        "<br><br>" +
+        "The competitor bot chose " +
+        botChoice +
+        "." +
+        "<br><br>" +
+        result +
+        "<br><br>" +
+        "Now you can type 'play' to play another round!" +
+        "<br><br>" +
+        "Hey, " +
+        userName +
+        " you have played " +
+        gameRecord +
+        " games so far. Your win:loss:draw figures is " +
+        winRecord +
+        ":" +
+        lossRecord +
+        ":" +
+        drawRecord +
+        "." +
+        performanceMesssage +
+        "."
+      );
+    } //korean mode return
+    else if (gameType == "korean") {
+      if (koreanLastWin == "You won." && result == "It's a draw.") {
+        return (
+          "You won the last round..." +
+          "<br><br>" +
+          "This round is a draw...so..." +
+          "<br><br>" +
+          "According to the rules of Mumuk-jji-ppa, you win!"
+        );
+      } else if (koreanLastWin == "You lost." && result == "It's a draw.") {
+        return (
+          "You lost the last round..." +
+          "<br><br>" +
+          "This round is a draw...so..." +
+          "<br><br>" +
+          "According to the rules of Mumuk-jji-ppa, you lose..."
+        );
+      } else {
+        if (result == "You win.") {
+          koreanLastWin = "You won.";
+        } else if (result == "You lose.") {
+          koreanLastWin = "You lost.";
+        } else {
+          koreanLastWin = "nil";
+        }
+        return (
+          "You chose " +
+          userChoice +
+          "." +
+          "<br><br>" +
+          "The bot chose " +
+          botChoice +
+          "." +
+          "<br><br>" +
+          "For this round, " +
+          result +
+          "<br><br>" +
+          "Based on Mumuk-jji-ppa rules, there are no winners yet. Keep trying!"
+        );
+      }
+    }
+  }
```

Comment:
abstracting game functions to run in a helper function also reduces the nesting as seen here - which can cause problems later down the road if bugs starting popping up